MMLanScan/Misc/NetworkCalculator.m

```diff
@@ -158,30 +158,25 @@ +(BOOL)isValidIPAddress:(NSString*)ipAddress {
 
 //This function convert decimals to binary
 +(NSString *)print01:(int)int11{
+    int n = 128;
+    NSMutableString *str = [[NSMutableString alloc] init];
     
-    int n =128;
-    char array12[8];
```

Comment:
Alternatively, you can make `char array12[9]` size 9 and make the last char null `'\0'`